backend/app/services/kind_impl.py

```
@@ -697,3 +697,76 @@ def _format_resource(self, resource: Kind) -> Dict[str, Any]:
+        """Ensure only one device can be default per user"""
+        from sqlalchemy import and_
+
+        # Find all other Device CRDs for this user that are default
+        devices = (
+            db.query(Kind)
+            .filter(
+                and_(
+                    Kind.user_id == user_id,
+                    Kind.kind == "Device",
+                    Kind.namespace == "default",
+                    Kind.is_active == True,
```

Ruff (0.14.14)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)